subjects/devops/backup_manager/audit/README.md

```diff
@@ -0,0 +1,135 @@
+
+##### Create a couple more backup schedules using the command `python3 ./backup_manager.py create "file;HH:MM;backup_name"`
+
+##### Run the command `python3 ./backup_manager.py list`.
+
+```bash
+$ python3 ./backup_manager.py list
+0: test2;18:15;backup_test2
+1: test3;18:16;backup_test3
+2: test4;18:17;backup_test4
+```
+
+###### Did you get a result like the one above, with the schedule you created attached with an index at the beginning, on your terminal?
+
+##### Run the command `python3 ./backup_manager.py delete 1`.
+
+```bash
+$ python3 ./backup_manager.py list
+0: test2;18:15;backup_test2
+1: test4;18:17;backup_test4
+```
+
+###### Was the task with the index `1` removed from the list? Use `python3 ./backup_manager.py list`to.
+
+##### Verify that the following command runs without errors: `python3 backup_manager.py start`.
+
+###### Can you confirm that the `backup_service.py` process is running by using the command `ps -ef | grep backup_service`?
+
+##### Now run the command `python3 backup_manager.py start` again.
+
+```bash
+$ python3 ./backup_manager.py start
+$ cat logs/backup_manager.log
+[13/02/2023 17:14] Error: service already running
+```
+
+###### Can you confirm bu running the command `cat logs/backup_manager.log` if the log file contains an error like the one above, stating that the service is already running?
+
+##### Run `python3 backup_manager.py stop` and delete everything that was created until now, the `logs` folder and the `backup_schedules.txt`. Lets create the backup manually.
+
+##### Run the following command by order:
+
+    1- mkdir testing; cd testing; touch 1 2 3; cd ../
+    2- python3 ./backup_manager.py create "testing;"Current_hour";backup_test".
+    3- python3 ./backup_manager.py start
+    4- python3 ./backup_manager.py backups
+
+```bash
+$ mkdir testing; cd testing; touch 1 2 3; cd ../
+$ ls
+backup_manager.py  backup_service.py  testing/
+$ python3 ./backup_manager.py create "testing;18:21;backup_test"
+$ python3 ./backup_manager.py list
+0: testing;18:21;backup_test
+$ python3 ./backup_manager.py start
+$ python3 ./backup_manager.py backups
+backup_test.tar
+$ ls
+backup_manager.py  backups/  backup_schedules.txt  backup_service.py  logs/ testing/
+$ cat logs/backup_manager.log
+[13/02/2023 18:20] Schedule created
+[13/02/2023 18:20] Show schedule list
+[13/02/2023 18:21] Backup list
+$ cat backup_schedules.txt
+testing;18:21;backup_test
+$ cat logs/backup_service.log
+[13/02/2023 18:21] Backup done for testing in backups/backup_test.tar
+```
+
+##### Can you confirm that the backup was created successfully? Follow the example above.
+
+### Error handling
+
+##### Verify error handling for incorrect commands and incorrect arguments. For example, confirm that an error message is displayed when attempting to run the command `python3 backup_manager.py invalid_command`.
+
+```bash
+$ python3 backup_manager.py invalid_command
+Error: unknown instruction
+```
+
+###### Was the error message displayed?
```

Comment:
We should agree if this should be logged or printed on strout... Normally I think it should be logged (so not `displayed`) in the log file as everything else.

In general I guess the only think that should be printed is the error related to not being able to create/write in the log file.
It's ok if the student wants also to print something on stdout when an operation is done, but this is optional, what is mandatory in my opinion is to log in the file.

Reply:
Ok, for me its fine but we need to recheck the Readme to see if we didn't ask them to print

Reply:
The subject is normally fixing rules general enough to enforce this behavior:
"All actions and errors should be logged into a specific file in the `./log` directory."

I can maybe add a small tip at the end of that paragraph in the subject saying if they want they can also write on stdout, tough only the logs in the files are mandatory.